inference-engine/samples/common/samples/common.hpp

```diff
@@ -1118,6 +1118,18 @@ inline std::size_t getTensorBatch(const InferenceEngine::TensorDesc& desc) {
     return 0;
 }
 
+inline void setBatchSizeUsingReshape(InferenceEngine::CNNNetwork& network, size_t batch) {
+    InferenceEngine::ICNNNetwork::InputShapes inputShapes = network.getInputShapes();
+    for (auto& shape : inputShapes) {
```

Comment:
I think it is not correct for common case. Because network can have several inputs and some inputs don't have a batch dimension.